src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1771,9 +1821,14 @@ void Interpolate::initSupportedPrimitiveDescriptors() {
                     pushDesc(LayoutType::nCsp8c, jit_sse42);
             }
         }
-
+        const auto &dataMaxDims = getInputShapeAtPort(DATA_ID).getMaxDims();
         // planar for 1.ref on machine without sse41(if no sse41, canFuse() is false). 2.JIT kernel for f32 && avx2(gather).(with fuse)
-        if (mayiuse(cpu::x64::avx2) && inputPrecision == Precision::FP32) {
+        const bool allowAvx2NcspFp32 = mayiuse(cpu::x64::avx2) && inputPrecision == Precision::FP32;
+        // allow I8 planar format only for avx2 with channels = 1, fusing is not supported
+        const bool allowAvx2NcspI8 = mayiuse(cpu::x64::avx2) && !mayiuse(cpu::x64::avx512_core) &&
```

Comment:
why not support sse41 and avx512 target, the emulation should and does not have such limitations.